nexus/src/authz/omicron.polar

```diff
@@ -230,8 +230,29 @@ has_relation(fleet: Fleet, "parent_fleet", collection: ConsoleSessionList)
 # read silo users and modify their sessions.  This is necessary for login to
 # work.
 has_permission(actor: AuthenticatedActor, "read", user: SiloUser)
-	if has_role(actor, "external-authenticator", user.fleet);
+	if has_role(actor, "external-authenticator", user.silo.fleet);
 has_permission(actor: AuthenticatedActor, "read", session: ConsoleSession)
 	if has_role(actor, "external-authenticator", session.fleet);
 has_permission(actor: AuthenticatedActor, "modify", session: ConsoleSession)
 	if has_role(actor, "external-authenticator", session.fleet);
+
+resource SiloUser {
+	permissions = [ "read", "modify", "create_child" ];
+        relations = { parent_silo: Silo };
+
+        "read" if "read" on "parent_silo";
```

Comment:
Huh -- I always thought the second string here had to be a _role_, but I see from [the docs](https://docs.osohq.com/any/reference/polar/polar-syntax.html#shorthand-rules) that it's allowed to be a permission, too.  I wonder if we should change the existing shorthand rules to use permissions instead of roles, or use roles here for consistency, or not sweat the difference.

Reply:
Roles added in 95114e05.